framework/src/utils/MultiMooseEnum.C

```diff
@@ -78,36 +70,45 @@ MultiMooseEnum::operator!=(const MultiMooseEnum & value) const
 bool
 MultiMooseEnum::contains(const std::string & value) const
 {
-  std::string upper(value);
-  std::transform(upper.begin(), upper.end(), upper.begin(), ::toupper);
-
-  return std::find(_current_names.begin(), _current_names.end(), upper) != _current_names.end();
+  std::string upper(MooseUtils::toUpper(value));
```

Comment:
See the usage of this new utility is copy heavy. Copy the input and copy the output.